src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -276,6 +330,8 @@ function loadOperation(
     const summary = getSummary(program, op);
     const externalDocs = getExternalDocs(program, op);
 
+    const consumes = getConsumes(program, op);
+    const produces = getProduces(program, op);
```

Comment:
I didn't see this 'produces' used in the code, where do we plan to use it?

Reply:
we will not support @consumes @produces as operation decorator. remove them.